tasks/run-matlab-build/v0/task.json

```diff
@@ -0,0 +1,34 @@
+{
+    "": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "c61c6fcc-00f1-4aa0-8513-31aad3340512",
+    "name": "RunMATLABBuild",
+    "friendlyName": "Run MATLAB Build",
+    "description": "Run a build using the MATLAB build tool. Use this task to run the MATLAB build tasks specified in a file named buildfile.m in the root of your repository. To use the run-build task, you need MATLAB R2022b or a later release.",
+    "helpMarkDown": "",
+    "category": "Build",
+    "author": "The MathWorks, Inc.",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "inputs": [
+        {
+            "name": "tasks",
+            "type": "string",
+            "label": "Tasks",
+            "required": false,
+            "defaultValue": "",
+            "helpMarkDown": "Space-separated list of tasks to run. If not specified, the task runs the default tasks in the file buildfile.m as well as all the tasks on which they depend."
+        }
+    ],
+    "instanceNameFormat": "Run MATLAB Build",
+    "execution": {
+        "Node10": {
+            "target": "main.js"
+        }
+    },
+    "messages": {
+        "BuildFailed": "Build failed with non-zero exit code."
```

Comment:
Could we remove the hyphen here?

https://www.mathworks.com/support/search.html?q=nonzero&fq[]=asset_type_name:documentation&fq[]=category:matlab/index&page=1

Reply:
fixed